python/src/common.cpp

```diff
@@ -65,6 +65,11 @@ void common(py::module& m)
       .def_property_readonly("ghosts", &dolfin::common::IndexMap::ghosts,
                              py::return_value_policy::reference_internal,
                              "Return list of ghost indices")
+      .def_property_readonly("shared_indices",
+                             &dolfin::common::IndexMap::shared_indices,
+                             py::return_value_policy::reference_internal,
```

Comment:
This doesn't like right since the function returns a value rather than reference or pointer.